api/hooks/discordBot/commands/sdtd/top.js

```diff
@@ -13,7 +13,9 @@ class Top extends Commando.Command {
         key: 'type',
         prompt: 'Please specify which top you want to see',
         type: 'string',
-        oneOf: ['currency', 'zombies', 'players', 'deaths', 'playtime', 'score', 'level']
+        oneOf: ['currency', 'zombies', 'players', 'deaths', 'playtime', 'score', 'level'],
+        validate: (val, msg, arg) => arg.oneOf.some(valid => val === valid),
+        error: 'Top Command: Type argument value was not valid.'
```

Comment:
Would it be possible to include a list of valid args here? Can reuse the `oneOf` array for this.

Reply:
Are you talking about the error message? I can hardcode the valid commands or define the valid commands outside of the module exports

Reply:
I'd say define `['currency', 'zombies', 'players', 'deaths', 'playtime', 'score', 'level']` outside of the command and use it in the `oneOf` property and in the error message.

Something like 
```
error: `Top Command: Type argument value was not valid. Please use one of: ${validArgs.join(', ')}`
```

perhaps?